subsys/net/l2/ethernet/Kconfig

```diff
@@ -98,6 +98,14 @@ config NET_ARP_GRATUITOUS_INTERVAL
 	depends on NET_ARP_GRATUITOUS_TRANSMISSION
 	default 60
 
+config NET_ACCEPT_MISMATCH_L3_L2_ADDR
```

Comment:
IMHO this should be prefixed with `NET_L2_ETHERNET_` - the current name is too generic, might be confusing